src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -2280,6 +2280,11 @@ bool CacheManager::SetMetaInfo(const std::string& ccpu_version,
   return true;
 }
 
+const std::string& CacheManager::GetCCPUVersion() {
+  LOG4CXX_AUTO_TRACE(logger_);
+  return *pt_->policy_table.module_meta->ccpu_version;
```

Comment:
@ydementieiev Get first module meta from PT as a reference and then extract ccpu_version
```cpp
 rpc::Optional<policy_table::ModuleMeta>& module_meta =
      pt_->policy_table.module_meta;
  return *(module_meta->ccpu_version);
```

Reply:
@AByzhynar  [done](https://github.com/smartdevicelink/sdl_core/pull/3269/commits/0d60c4e63814fd53aa0e6c9e0f26893e9587efca#diff-364146db2d1780b4286234cb8c7db37dR2287)